Modules/_interpretersmodule.c

```diff
@@ -1533,18 +1504,21 @@ interp_decref(PyObject *self, PyObject *args, PyObject *kwds)
 }
 
 
+/*[clinic input]
+_interpreters.capture_exception
+    exc_arg: object(c_default='NULL') = None
+    /
```

Comment:
The name is "exc", and it is not positional-only. Also, there is no need to use `c_default='NULL'`.
```suggestion
    exc: object = None
```

Reply:
Oh, this is a mistake. I think it should be `exc as exc_arg`. I originally used `exc` but there is a check under the `finally` label for `exc != exc_arg`.

```suggestion
    exc as exc_arg: object = None
```